CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 ## Unreleased
 
+### Added
+
+-   Factory for `Redirect` model in test app ([#380](https://github.com/torchbox/wagtail-grapple/pull/380)) @JakubMastalerz
```

Comment:
Did you try doing the below?

```suggestion
-   Factory for `Redirect` model in test app (!380) @JakubMastalerz
```

or

```suggestion
-   Factory for `Redirect` model in test app ([!380]) @JakubMastalerz
```

Reply:
Yes, didn't work unfortunately, so I reversed to the old method.